process/smartContract/process.go

```diff
@@ -394,18 +394,10 @@ func (sc *scProcessor) finishSCExecution(
 		return 0, err
 	}
 
-	if !sc.flagDeploy.IsSet() {
-		err = sc.updateDeveloperRewardsV1(tx, vmOutput, builtInFuncGasUsed)
-		if err != nil {
-			log.Error("updateDeveloper rewards error", "error", err.Error())
-			return 0, err
-		}
-	} else {
-		err = sc.updateDeveloperRewardsV2(tx, vmOutput, builtInFuncGasUsed)
-		if err != nil {
-			log.Error("updateDeveloper rewards error", "error", err.Error())
-			return 0, err
-		}
+	err = sc.updateDeveloperRewardsProxy(tx, vmOutput, builtInFuncGasUsed)
+	if err != nil {
+		log.Error("updateDeveloper rewards error", "error", err.Error())
```

Comment:
To avoid printing twice the word "error" you can replace this line with:
log.Error("updateDeveloperRewardsProxy", "error", err.Error())

Reply:
done